### PR TITLE
Add basic read function wrappers.

### PR DESCRIPTION
**Pull request by ueshin**
This PR adds a couple of basic read function wrappers of pandas' corresponding to `to_xxx()` wrappers:

- `read_clipboard()`
- `read_excel()`
- `read_html()`

### CODE REVIEW

File: databricks/koalas/namespace.py [hunk 1]
```diff
@@ -279,6 +280,373 @@ def read_parquet(path, columns=None):
...
+
+        If a column or index contains an unparseable date, the entire column or
+        index will be returned unaltered as an object data type. For non-standard
+        datetime parsing, use ``pd.to_datetime`` after ``pd.read_csv``
+
+        Note: A fast-path exists for iso8601-formatted dates.
+    date_parser : function, optional
+        Function to use for converting a sequence of string columns to an array of
+        datetime instances. The default uses ``dateutil.parser.parser`` to do the
+        conversion. Pandas will try to call `date_parser` in three different ways,
```
Comment (HyukjinKwon) [inline databricks/koalas/namespace.py]: nit: Pandas -> Koalas

File: databricks/koalas/namespace.py [hunk 2]
```diff
@@ -279,6 +280,373 @@ def read_parquet(path, columns=None):
...
+
+    See Also
+    --------
+    read_csv
+
+    Notes
+    -----
+    Before using this function you should read the `gotchas about the
+    HTML parsing libraries
+    <https://pandas.pydata.org/pandas-docs/stable/user_guide/io.html#io-html-gotchas>`__.
```
Comment (HyukjinKwon) [inline databricks/koalas/namespace.py]: nit: we can just remove the whole "Notes" (because of the pandas' link) I guess.

Comment (ueshin) [reply databricks/koalas/namespace.py]: sounds good.

**Comment by ueshin** [comment]
merging.

**Comment by naveenag86** [comment]
Using Koalas, is it possible to create excel and multiple worksheets? 
and also does it allow some kind of formatting like font size, color fill etc..?

**Comment by HyukjinKwon** [comment]
If it works in pandas, Koalas should. If that doesn't work, feel free to file an issue.

**Pull request merged**